storage/local/storage.go

```diff
@@ -1474,7 +1479,7 @@ func (s *MemorySeriesStorage) writeMemorySeries(
 		var offset int
 		offset, persistErr = s.persistence.persistChunks(fp, chunks)
```

Comment:
persistChunks's comment says it's the responsibility of the caller to quarantine the series, similarly with the others looking at their code.